xrdp/Makefile.am

```diff
@@ -98,6 +98,9 @@ dist_xrdppkgdata_DATA = \
   xrdp24b.bmp \
   xrdp256.bmp \
   xrdp_logo.bmp \
+  xrdp_logo_1.bmp \
```

Comment:
Replace the old logo. The old one can be resurrected from git history if necessary.

Reply:
Agreed.

Reply:
Roger.
I will replace the existing file.